### PR TITLE
refactor: create-react-app -> vite

### PR DESCRIPTION
**Pull request by robertoschiavone**

### CODE REVIEW

File: week3/possible-qa-exercises/2-react-testing-library-exercises/src/5-Context/logged-in-user-context.jsx [hunk 1]
```diff
@@ -1,31 +1,40 @@
...
   },
 });
 
 /**
- * We've put the Provider in the same file as the Context as the Context and Provider are interconnected.
- * The Provider is only needed once, so we make that a named export and have the Context be the default export.
+ * We've put the Provider in the same file as the Context as the Context and
+ * Provider are interconnected.
+ * The Provider is only needed once, so we make that a named export and have
+ * the Context be the default export.
```
Comment (tlorent) [inline week3/possible-qa-exercises/2-react-testing-library-exercises/src/5-Context/logged-in-user-context.jsx:35]: This comment was formatted a bit clearer before:

```suggestion
 * We've put the Provider in the same file as the Context as the Context and Provider are interconnected.
 * The Provider is only needed once, so we make that a named export and have the Context be the default export.
```

File: week3/possible-qa-exercises/2-react-testing-library-exercises/src/5-Context/my-profile.jsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+          >
+            Log out
+          </button>
+        </div>
+    );
+  } else {
+    // This is of course a HORRIBLE user experience, there would normally be a
+    // link to a login page or a login form directly.
+    // We are simplifying for educational purposes, we do NOT want to see things
+    // like this in your projects :D
```
Comment (tlorent) [inline week3/possible-qa-exercises/2-react-testing-library-exercises/src/5-Context/my-profile.jsx:30]:
```suggestion
/* 
  This is of course a HORRIBLE user experience, there would normally be a
  link to a login page or a login form directly.
  We are simplifying for educational purposes, we do NOT want to see things
  like this in your projects :D 
*/
```

File: week3/possible-qa-exercises/2-react-testing-library-exercises/src/5-Context/my-profile.jsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+function MyProfile() {
+  const {loggedInUser, isLoggedIn, logout} = useContext(LoggedInUserContext);
+
+  if (isLoggedIn) {
+    return (
+        <div>
+          <p>{`Welcome back ${loggedInUser.name}!`}</p>
+          {/* Here may be some information of the user, maybe some notifications.
+        Cut out for simplicity */}
```
Comment (tlorent) [inline week3/possible-qa-exercises/2-react-testing-library-exercises/src/5-Context/my-profile.jsx:15]:
```suggestion
          {/* Here may be some information of the user, maybe some notifications. Cut out for simplicity */}
```

File: week3/possible-qa-exercises/2-react-testing-library-exercises/src/5-Context/testids.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
 /**
- * This is the same idea as in the Routing example, have a look at that file for a big explanation of why we do this
+ * This is the same idea as in the Routing example, have a look at that file for
+ * a big explanation of why we do this
```
Comment (tlorent) [inline week3/possible-qa-exercises/2-react-testing-library-exercises/src/5-Context/testids.js:3]:
```suggestion
 * This is the same idea as in the Routing example, have a look at that file for a big explanation of why we do this
```

**Review by tlorent — COMMENTED** [review]
Thanks for the effort! Great that you're proactive with this! 🙌 

Looks like you have different Prettier settings or something? I see a lot of formatting changes. Specifically all the comments now look a bit strange. Can you perhaps have a look at that?

File: week1/prep-exercises/1-counter/README.md [hunk 5]
```diff
@@ -2,7 +2,7 @@
 
 The counter is one of the best way to learn about state management as it is nice and simple and isolates the management of state. Let's create a simple number counter, starting from 0 until infinity!
 
-1. Run `create-react-app` in this folder to create your environment
+1. Run `vite` in this folder to create your environment
```
Comment (stasel) [inline week1/prep-exercises/1-counter/README.md:5]: Should this be `npm create vite`?

File: week2/prep-exercises/1-generate-random-person-details/README.md [hunk 6]
```diff
@@ -8,7 +8,7 @@ A common problem is to know where to start!
 
 ## 1. Setup
 
-1. First, let's make sure we have some place to code. Run `create-react-app` in this folder to create your environment
+1. First, let's make sure we have some place to code. Run `npm create vite assignment -- --template react` in this folder to create your environment
```
Comment (stasel) [inline week2/prep-exercises/1-generate-random-person-details/README.md:11]: Is the `--` in the middle required here? Or was it a type?

**Review by stasel — APPROVED** [review]
Awesome work! Thanks for your contribution!

File: week2/prep-exercises/1-generate-random-person-details/README.md (hunk 6 above)
Comment (robertoschiavone) [reply week2/prep-exercises/1-generate-random-person-details/README.md:11]: It is explicitly mentioned in the [Getting Started](https://vitejs.dev/guide/#scaffolding-your-first-vite-project)

File: week3/possible-qa-exercises/2-react-testing-library-exercises/src/5-Context/logged-in-user-context.jsx (hunk 1 above)
Comment (robertoschiavone) [reply week3/possible-qa-exercises/2-react-testing-library-exercises/src/5-Context/logged-in-user-context.jsx]: I've left the break at 80 chars but I added spacing to improve readability

File: week3/possible-qa-exercises/2-react-testing-library-exercises/src/5-Context/my-profile.jsx (hunk 2 above)
Comment (robertoschiavone) [reply week3/possible-qa-exercises/2-react-testing-library-exercises/src/5-Context/my-profile.jsx]: Done! All comments now are multiline comments

File: week3/possible-qa-exercises/2-react-testing-library-exercises/src/5-Context/my-profile.jsx (hunk 3 above)
Comment (robertoschiavone) [reply week3/possible-qa-exercises/2-react-testing-library-exercises/src/5-Context/my-profile.jsx]: I've left the break at 80 chars but I added spacing to improve readability

File: week3/possible-qa-exercises/2-react-testing-library-exercises/src/5-Context/testids.js (hunk 4 above)
Comment (robertoschiavone) [reply week3/possible-qa-exercises/2-react-testing-library-exercises/src/5-Context/testids.js:3]: Same as above

File: week1/prep-exercises/1-counter/README.md (hunk 5 above)
Comment (robertoschiavone) [reply week1/prep-exercises/1-counter/README.md:5]: Yep, fixed!

**Review by stasel — APPROVED** [review]

**Comment by robertoschiavone** [comment]
@stasel I cannot merge it, I do not have write access

**Pull request merged**